docs/ref/contrib/admin/index.txt

```diff
@@ -1855,10 +1855,11 @@ templates used by the :class:`ModelAdmin` views:
     unauthorized data exposure.
 
     The ``lookup_allowed()`` method is given a lookup path from the query string
-    (e.g. ``'user__email'``) and the corresponding value
-    (e.g. ``'user@example.com'``), and returns a boolean indicating whether
-    filtering the changelist's ``QuerySet`` using the parameters is permitted.
-    If ``lookup_allowed()`` returns ``False``, ``DisallowedModelAdminLookup``
+    (e.g. ``'user__email'``), the corresponding value
+    (e.g. ``'user@example.com'``) and the request, and returns a boolean
```

Comment:
Comma is missing before 'or'/'and' (see https://en.wikipedia.org/wiki/Serial_comma).
```suggestion
    (e.g. ``'user@example.com'``), and the request, and returns a boolean
```